lib/Cake/Utility/ObjectCollection.php

```diff
@@ -95,6 +95,7 @@ public function trigger($callback, $params = array(), $options = array()) {
 		if (empty($this->_enabled)) {
 			return true;
 		}
+		$subject = null;
```

Comment:
Tests fail here too.

Very similar problem.

---

To CakePHP Team: Why Travis tests are not running on `2.x`?

Reply:
> To CakePHP Team: Why Travis tests are not running on 2.x?

I'm not sure. Travis used to run tests for 2.x Its possible that there is a problem with the configuration file, or perhaps we're trying to use an unsupported worker type.

Reply:
It looks like travis ran tests for [2.x 8 days ago](https://travis-ci.org/cakephp/cakephp/builds/416046280) I'm not sure why it isn't running this pull request.

Reply:
Can we merge this as this seems to be a Travis issue?

Reply:
Tests fail here, so should be fixed

Reply:
@garas Where can I see the failure?

Reply:
[Run tests](https://book.cakephp.org/2.0/en/development/testing.html#running-tests):

`lib/Cake/Console/cake test core AllTests`

You can also run only some test cases like `Utility/ObjectCollection` or `Network/CakeResponse`

Reply:
I cannot install PHPUnit 3.7, as I already have 7.3 installed on my machine. Can you help me here, by showing me the issue? Could not find it in the PR tools

Reply:
You can install phpunit in CakePHP dir with `composer require --dev phpunit/phpunit:"3.7.38"`

Reply:
Pushed a fix, and now both tests are passing